doc/source/whatsnew/v0.22.0.txt

```diff
@@ -24,7 +24,8 @@ Other Enhancements
 
 - Better support for :func:`Dataframe.style.to_excel` output with the ``xlsxwriter`` engine. (:issue:`16149`)
 - :func:`pandas.tseries.frequencies.to_offset` now accepts leading '+' signs e.g. '+1h'. (:issue:`18171`)
--
+- :class:`pandas.io.formats.style.Styler` now has ``index`` parameter and corresponding method ``hide_index()`` to determine whether the index will be rendered in ouptut (:issue:`14194`)
+- :class:`pandas.io.formats.style.Styler` now has ``hidden_cols`` parameter and corresponding method ``hide_columns()`` to determine whether columns will be hidden in output (:issue:`14194`)
```

Comment:
hide_cols -> hide_columns

Reply:
didn't we eliminate the paramaters as user facing and only use the methods? (which i think is better anyhow)

Reply:
@jreback yes, thanks, i should have caught that. fixed!